src/main/java/uk/ac/cam/cl/dtg/isaac/dos/eventbookings/EventBookings.java

```diff
@@ -180,4 +190,4 @@ Iterable<EventBooking> findAllByEventIdAndStatus(String eventId, @Nullable Booki
    * @param userId - user id
    */
   void deleteAdditionalInformation(Long userId) throws SegueDatabaseException;
-}
+}
```

Comment:
You can configure your IDE so that it always ensures that the files end with an EOL character. To be honest, I would have expected that to be caught by checkstyle.